.github/pull_request_template.md

```diff
@@ -0,0 +1,23 @@
+## Description
+<!-- Include a summary of the change made and also list the dependencies that are required if any -->
+
+---
+## Issue Ticket Number
+Fixes [Issue #]
+
+---
+## Type of change
+<!-- Please select all options that are applicable. -->
+- [ ] Bug fix (non-breaking change which fixes an issue)
+- [ ] New feature (non-breaking change which adds functionality)
+- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
+- [ ] This change requires a documentation update
+
+---
+# Checklist:
```

Comment:
can we add do this checklist Tests? there's not a testing suite yet but that's being added in another active Issue

Reply:
@colbyfayock 
To understand correctly, you are requesting that we change the current title from "checklist" to "do this checklist tests"

OR 

Add a checklist item where one is supposed to run a specific test (which is an active issue being worked on)

Reply:
#2 - add a checklist item about testing, i was thinking more about ADDING tests, but running tests make sense too

Reply:
@colbyfayock I've made the changes as requested, feel free to review the latest commit